reference/events.v1.yaml

```diff
@@ -28,7 +32,7 @@ tags:
 paths:
   /events:
     get:
-      description: Allows a subscriber to retrieve a list of past events.
+      description: Allows a subscriber to retrieve a list of past events. The provider will not keep the events indefinate. This API is intended when a party had downtime and needs to catchup on processing/receiving events. The intended retention is a few days to support catching up.
```

Comment:
Agree - but does this mean that we somehow should be able to retrigger the loading of initial data (imagine the situation where a party has been offline for just too long and therefor is not able to catch up)

Reply:
Yes, Doing something in the consent protocol would be usefull (reconsent could trigger the initial flow?)

Reply:
Create issue: https://github.com/stichtingsem/ecosystem/issues/35